examples/mxnet/gat/gat_batch.py

```diff
@@ -212,9 +247,10 @@ def main(args):
             help="input feature dropout")
     parser.add_argument("--attn-drop", type=float, default=.6,
             help="attention dropout")
-    parser.add_argument("--lr", type=float, default=0.005,
+    parser.add_argument("--lr", type=float, default=0.01,
```

Comment:
lr is different from (larger than) pytorch version. Is it because the gradent rescale?